bin/node/runtime/Cargo.toml

```diff
@@ -116,6 +116,13 @@ substrate-wasm-builder = { version = "5.0.0-dev", path = "../../../utils/wasm-bu
 default = ["std"]
 with-tracing = ["frame-executive/with-tracing"]
 std = [
+	"sp-sandbox/std",
```

Comment:
You can just remove the unused depedency, then we don't need to add it here :P

Reply:
There also is [cargo udeps](https://github.com/est31/cargo-udeps) which claims to find unused crates.  
I will give it a try later.